.github/workflows/ci.yml

```diff
@@ -1317,40 +1563,37 @@ jobs:
         else
           CPLUS_INCLUDE_PATH="${LLVM_DIR}/llvm/include:${LLVM_DIR}/clang/include:${LLVM_BUILD_DIR}/include:${LLVM_BUILD_DIR}/tools/clang/include:$PWD/include"
         fi
-
+        git apply -v ./patches/llvm/CppInterOp-temp-testing.patch
         # Build CppInterOp next to cling and llvm-project.
         mkdir build
         cd build
         export CPPINTEROP_BUILD_DIR=$PWD
-
         if [[ "${cling_on}" == "ON" ]]; then
           emcmake  cmake -DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }}        \
                 -DUSE_CLING=ON                                  \
                 -DUSE_REPL=OFF                                  \
-                -DCMAKE_PREFIX_PATH=$PREFIX                     \
+                -DCMAKE_PREFIX_PATH="$PREFIX"                     \
                 -DCling_DIR=$LLVM_BUILD_DIR/tools/cling         \
                 -DLLVM_DIR=$LLVM_BUILD_DIR/lib/cmake/llvm       \
+                -DLLD_DIR=$LLVM_BUILD_DIR     \
```

Comment:
Hmm as we are interested in `lld` 
I would just use 
```
-DLLVM_DIR=$LLVM_BUILD_DIR/lib/cmake/lld
```

Reply:
@anutosh491 This is a change for your PR on the shared library.

Reply:
Cool !